### ISSUE TITLE
Laslib vs PDAL vs Liblas

### ISSUE THREAD

**Issue by Gerbrand-V** [tudelft3d/3dfier#109]
At present Lastools/Laszip (laslib) is used. Some parts of Lastools have a commercial license attached to it, so some care should be taken which parts you compile and use. Futhermore: compilation from source of a 64bit version on Windows (VS2017) requires quite some effort (lots of compiler directives + warning ignores + redefinition of certain types that are used in the source). Is PDAL (BSD license) not more practical or better suited for use in 3dfier? Or Liblas (can be installed through OSGeo4W)?

Furthermore: GPL2.1 (as used by Laslib) requires an explicit statement that Laslib is used in the creation of the binary version (https://github.com/tudelft3d/3dfier/releases/latest). This  might also be the case for other libraries as well. The binary install does not contain such a notice and the 3dfier --license does not mention the GPL libraries that were used to create the executable. Maybe the used libraries + appropriate licenses could be mentioned there or in a separate text-document in the zip-file.

**Comment by tcommandeur** [comment]
Previous versions of 3dfier used Liblas but it is not beeing actively maintained for many years and has quite some bugs/issues we found. That is the reason we switched to Laslib (Lastools) since the library itself is open source and allowed to be used in thirdparty projects. As for PDAL, we did think about it before since it also allows reading of other point types and even raster files. However the similarity to the Liblas API and performance of Laslib made us choose that over PDAL.

We should in --license describe the used libraries that is a good idea, also if its not needed according to the license itself. As for the explicit statement in the binary, where does it need to be stated? I think in the zip file itself in a text file would be appropriate.
